cmd/backfillEvents.go

```diff
@@ -50,7 +51,11 @@ func backFillEvents() error {
 	blockChain := getBlockChain()
 	db := utils.LoadPostgres(databaseConfig, blockChain.Node())
 
-	extractor := logs.NewLogExtractor(&db, blockChain)
+	repo, repoErr := repositories.NewCheckedHeadersRepository(&db, genConfig.Schema)
+	if repoErr != nil {
+		return fmt.Errorf("error creating checked headers repository %w", repoErr)
```

Comment:
I _think_ this is probably a non-issue since we'll be running one container per schema, but maybe it'd be helpful to include the schema name in this formatted error?

Reply:
it should be in the error message, but it also shouldn't ever really happen so it won't hurt to stick in as well.